File: src/main/java/com/example/java_practice/commons/dto/Award.java
package com.example.java_practice.commons.dto;

import lombok.*;

/**
 * f_work_no 작품번호
 * f_title 제목
 * f_author 작가
 * f_year 출품년도
 * f_contest 공모전
 * f_award 상권명
 * f_code 작품코드
 * f_manager 주관
 * f_host 주최
 * f_reg_date 등록일시
 * f_filepath 파일경로
 * f_user_no 등록자 번호
 * f_work_size 사이즈
 * f_filename 파일명
 * f_memo 회원메모
 */
@Getter
@Setter
@NoArgsConstructor
@ToString
public class Award {
    private int f_work_no;
    private String f_title;
    private String f_author;
    private String f_year;
    private String f_contest;
    private String f_award;
    private String f_code;
    private String f_manager;
    private String f_host;
    private String f_reg_date;
    private String f_filepath;
    private int f_user_no;
    private String f_work_size;
    private String f_filename;
    private String f_memo;
}

File: src/main/resources/static/js/award.js
import * as util from './utils.js';

const Award = {
    init() {
        this.path = window.location.pathname.split('/').pop();
        util.bindEvents(this);
        util.initDatePicker(".flatpickr");
    },

    loadWork(workNo) {
        util.getAJAX('/api/admin/searchWork', { sort: this.path, workNo }, response => {
            this.fillWorkForm(response[0]);
        });
    },

    fillWorkForm(work) {
        $('#work-code').text(work.f_code);
        $('#work-title').val(work.f_title);
        $('#work-contest').val(work.f_contest);
        $('#work-author').val(work.f_author);
        $('#work-award').val(work.f_award);
        $('#work-host').val(work.f_host);
        $('#work-manager').val(work.f_manager);
        $('#work-year').val(work.f_year);
        $('#work-regdate').text(work.f_reg_date);
        $('#work-name').text(`${work.f_name}(${work.f_id})`);
        $('#work-dept').text(work.f_dept);
        $('#work-img').attr('src', work.f_filepath);

        $('#user-area').val(work.f_area);
        $('#user-name').val(work.f_name);
        $('#user-birth').val(work.f_birth);
        $('#user-phone').val(work.f_phone);
        $('#user-email').val(work.f_email);
        $('#sample4_roadAddress').val(work.f_main_address);
        $('#sample4_extraAddress').val(work.f_sub_address);
        $('#info-userNo').val(work.f_user_no);

        if(work.f_filepath) {
            $('.img-div, .download-btn').show();
        } else {
            $('.img-div, .download-btn').hide();
        }
    },

    fillAddress(result) {
        $('#sample4_postcode').val(result.zonecode);
        $('#sample4_roadAddress').val(result.roadAddr);
        $('#sample4_extraAddress').val(result.extraRoadAddr);
    }
};

document.addEventListener('DOMContentLoaded', () => Award.init());
